### PR TITLE
feat!: update to new db schema (MAPCO-4232, MAPCO-4290, MAPCO-5080 )

### PR DESCRIPTION
**Pull request by RonitKissis**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✔                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✔                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/DAL/migration/fullSchema.sql [hunk 1]
```diff
@@ -90,27 +89,27 @@ CREATE INDEX ix_product_sub_type
...
-    (source_start_date ASC NULLS LAST);
+    (imaging_time_begin_utc ASC NULLS LAST);
 
 -- Index: ix_source_end_date
 -- DROP INDEX ix_source_end_date;
-CREATE INDEX ix_source_end_date
+CREATE INDEX ix_imaging_time_end_utc
     ON records USING btree
-    (source_end_date ASC NULLS LAST);
+    (imaging_time_end_utc ASC NULLS LAST);
```
[the remark this answers is not in the document]
Comment (RonitKissis) [reply src/DAL/migration/fullSchema.sql:112]: added index to min_resolution_meter ( max_resolution_meter has one) - the other fields didnt have indexes

**Review by asafMasa — APPROVED** [review]

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: .github/workflows/build_and_push.yaml [hunk 2]
```diff
@@ -19,56 +19,9 @@ permissions:
...
-          login-server: ${{ secrets.ACR_URL }}
-          username: ${{ secrets.ACR_PUSH_USER }}
-          password: ${{ secrets.ACR_PUSH_TOKEN }}
-      - name: downcase repository
-        run: |
-          echo "Docker_Repository=${Docker_Repository,,}" >>${GITHUB_ENV}
-      - run: docker build . -t ${{ env.Docker_Repository }}
-      - run: docker push ${{ env.Docker_Repository }}
-
+      uses: MapColonies/shared-workflows/.github/workflows/build-and-push-docker.yaml@v1
```
Comment (asafMasa) [inline .github/workflows/build_and_push.yaml:22]: v1 or master?

File: .github/workflows/build_and_push.yaml [hunk 3]
```diff
@@ -19,56 +19,9 @@ permissions:
...
-      - name: save helm chart to local registry
-        run: |
-          helm chart save . '${{ secrets.ACR_URL }}/helm/${{ env.CHART }}:${{ env.VER }}'
-        working-directory: ./helm
-      
-      - name: publish chart to acr
-        run: |
-          helm chart push ${{ secrets.ACR_URL }}/helm/${{ env.CHART }}:${{ env.VER }}
-        working-directory: ./helm
+      uses: MapColonies/shared-workflows/.github/workflows/build-and-push-helm.yaml@v1
```
Comment (asafMasa) [inline .github/workflows/build_and_push.yaml:26]: v1 or master?

File: .github/workflows/release-on-tag-push.yaml [hunk 4]
```diff
@@ -6,23 +6,6 @@ on:
...
-
-      - name: Publish Release to Github
-        uses: softprops/action-gh-release@v1
-        with:
-          body_path: CHANGELOG.md
-          files: dump-server-postman-collection.json
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  release_on_tag_push:
+    uses: MapColonies/shared-workflows/.github/workflows/release-on-tag-push.yaml@v1
```
Comment (asafMasa) [inline .github/workflows/release-on-tag-push.yaml:10]: v1 or master?

File: .github/workflows/release-on-tag-push.yaml [hunk 5]
```diff
@@ -6,23 +6,6 @@ on:
...
+    secrets: inherit
```
Comment (asafMasa) [inline .github/workflows/release-on-tag-push.yaml:11]: end of file * 3 workflows

File: package.json [hunk 6]
```diff
@@ -109,4 +109,4 @@
     "ts-jest": "^29.0.5",
     "typescript": "^5.3.3"
   }
-}
+}
```
Comment (asafMasa) [inline package.json:112]: EOF

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: .github/workflows/release-on-tag-push.yaml (hunk 5 above)
Comment (RonitKissis) [reply .github/workflows/release-on-tag-push.yaml]: done

File: package.json (hunk 6 above)
Comment (RonitKissis) [reply package.json]: done

**Review by asafMasa — APPROVED** [review]

**Pull request merged**